maven-core/src/main/java/org/apache/maven/project/MavenProject.java

```diff
@@ -945,13 +945,21 @@ public void addAttachedArtifact( Artifact artifact )
         }
     }
 
+    /**
+     * Returns a mutable list of the attached artifacts to this project. It is highly advised <em>not</em>
+     * to modify this list, but rather use the {@link MavenProjectHelper}.
+     * <p>
+     * <strong>Note</strong>: This list will be made read-only Maven 4.
+     *
```

Comment:
You should close the HTML p tag, especially since all other HTML tags are properly closed.

Reply:
Good catch.